Gamma_Analysis.py

```diff
@@ -418,12 +420,15 @@ def check_spectra(samples, background, reference):
                             error_spectrum)
 
 
-def main():
-    background = SPEFile.SPEFile("USS_Independence_Background.Spe")
+def main(background_filename=BACKGROUND, reference_filename=REFERENCE, 
+         file_list=None):
+    
+    background = SPEFile.SPEFile(background_filename)
     background.read()
-    reference = SPEFile.SPEFile("UCB018_Soil_Sample010_2.Spe")
+    reference = SPEFile.SPEFile(reference_filename)
     reference.read()
-    sample_measurements, sample_names = acquire_files()
+    if file_list is None:
+        sample_measurements, sample_names = acquire_files()
```

Comment:
where is calibrate.get_sample_names located? As far as I know there is no get_sample_names_names method inside calibrate.py